deploy/digitalocean/scripts/marketplace-submit.sh

```
@@ -0,0 +1,36 @@
+EOF
+
+cat update.json
+
+echo ${ALBYHUB_VERSION}
+
+echo https://api.digitalocean.com/api/v1/vendor-portal/apps/${DIGITALOCEAN_APP_ID}
+
+curl --fail-with-body -X PATCH -H "Content-Type: application/json" -H "Authorization: Bearer ${DIGITALOCEAN_API_TOKEN}" \
+  -d @update.json  https://api.digitalocean.com/api/v1/vendor-portal/apps/${DIGITALOCEAN_APP_ID}
+
+if [ $? -eq 0 ]
```

Shellcheck (0.11.0)

[style] Check exit code directly with e.g. 'if mycmd;', not indirectly with $?.

(SC2181)